### ISSUE TITLE
How to get this library to work with your VPN_MQTT

### ISSUE THREAD

**Issue by garudaonekh** [levkovigor/ppposclient#1]
Hello,
I got this library to work with TTGO T-Call SIm800L. My purpose is to use it with your https://github.com/levkovigor/MQTT_VPN_ESP32/tree/main/mqtt_vpn_webserver

The missing piece here is this library is integrated with PubSubClient while your **mqtt_vpn_webserver** use ESP-MQTT.

How can we make this one integrated with ESP-MQTT?

Thanks;

**Comment by levkovigor** [comment]
Hi @garudaonekh,

PPPOSClient is my library, it should work fine. I've tested it and used already in some projects. But mqtt_vpn_webserver is not my library as I said earlier. I've only adapted it for Arduino IDE. I did not fully review the main algorithm of code and did not test it in detail. At the moment, I don't have time to deal with it. According to the tests that I conducted, I realized that VPN through MQTT is not the best idea for use in final products and not only in the case of ESP32. Maybe through another protocol, but not MQTT. Therefore, I advise you not to do this.
